users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -799,39 +804,35 @@ def run_scf_specaug_sort():
 
     nn_args, report_args_collection = get_nn_args_baseline(
         nn_base_args={
-            "bs2x5k_scf_specaugsortlayer2": dict(
-                returnn_args={**returnn_args, "specaug_old": {"max_feature": 15, "sort_layer2": True}},
+            "baseline": dict(
+                returnn_args={
+                    **base_returnn_args,
+                    "specaug_config": {"enable_sorting": False, "max_feature": 15, "steps_per_epoch": 4100},
+                },
                 feature_args=feature_args,
                 lr_args=lr_args,
-                report_args={"batch_size": "2x5k"},
+                report_args={"batch_size": "2x5k", "enable_sorting": False, "max_feature": 15},
             ),
-            "bs2x5k_scf_specaugsortlayer2frome210": dict(
+            "baseline_increase_flag": dict(
                 returnn_args={
-                    **returnn_args,
-                    "specaug_old": {"max_feature": 15, "sort_layer2": True},
-                    "extra_args": {
-                        "watch_memory": True,
-                        "conv_pad_seq_len_to_power": 1.5,
-                        "preload_from_files": {
-                            "existing-model": {
-                                "filename": (
-                                    "/work/asr4/vieting/setups/swb/work/20230406_feat/i6_core/returnn/training/"
-                                    "ReturnnTrainingJob.y9otnVMrBAWw/output/models/backup.epoch.210"
-                                ),
-                                "init_for_train": True,
-                            },
-                        },
-                        **returnn_args["extra_args"],
+                    **base_returnn_args,
+                    "specaug_config": {
+                        "enable_sorting": False,
+                        "max_feature": 15,
+                        "max_feature_num": 2,
+                        "steps_per_epoch": 4100,
+                        "time_mask_num_schedule": {0: 1, 1: 2},
```

Comment:
This does not make sense, does it? You should reduce it to half in the first epoch, but instead you double it from the second. I think it's anyway not possible to replicate the old behavior with the current implementation because the division by 2 is after the max with the sequence length dependent part. But maybe let's try if it works like this (without doubling like you do here) and only if not replicate in more detail.

Reply:
I reduced the start point and double it to the previous value. for the time dimension it is as you say, there is no way to replicate the old behavior. this is the closest that is possible.

Reply:
Why is this the closest possible? Why this `time_mask_num_schedule`? Shouldn't it be without? Maybe it does not make a difference, but this factor of two here does not make sense to me.

Reply:
I did not realize that I still had the second scheduling in there. It should only be the one for the features. I forgot to remove it after I realized that I could not replicate the behavior.